demisto_sdk/commands/content_graph/parsers/incident_field.py

```diff
@@ -22,6 +22,9 @@ def __init__(
         super().__init__(path, pack_marketplaces, git_sha=git_sha)
         self.field_type = self.json_data.get("type")
         self.associated_to_all = self.json_data.get("associatedToAll")
+        self.content = self.json_data.get("content", True)
+        self.system = self.json_data.get("system", False)
+        self.group = self.json_data.get("group", 0)
```

Comment:
```suggestion
        self.content = self.json_data.get("content", True)
        self.system = self.json_data.get("system", False)
        self.group = self.json_data.get("group", 0)
```
if it's missing from the yml, won't these defaults always pass validations?